packages/web-components/src/components/va-on-this-page/va-on-this-page.css

```diff
@@ -1,4 +1,5 @@
 @import "~@department-of-veterans-affairs/css-library/dist/stylesheets/uswds-typography.css";
+@import "~@department-of-veterans-affairs/css-library/dist/tokens/css/variables.css";
```

Comment:
Isn't this already loaded in the global stylesheet?

https://github.com/department-of-veterans-affairs/component-library/blob/4c02937c4642cab0aac16e5e5c233dd3fcb2588c/packages/web-components/src/global/main.css#L1

Reply:
Ah! Right you are! Removing this import now.